lib/remote_files/configuration.rb

```diff
@@ -103,6 +103,7 @@ def store_once!(file)
           break
         rescue ::RemoteFiles::Error => e
           file.logger.info(e) if file.logger
+          file.errors.push(e) if file.errors
```

Comment:
Let's make this nil safe: `file&.errors`

Reply:
I tried to add the nil safe operator but it didn't pass the  Ruby 2.2 tests. I think the operator in itself isn't used in ruby 2.2

Reply:
we should remove ruby 2.2 support I think 😅 (whether or not it should be part of this is a different question though)

Reply:
It tested correctly when I ran `bundle exec rake` locally but it kept failing the github `CI / Ruby 2.2 - bundle exec rake (push) `

Reply:
no need for a safe navigation here. The line above would fail if file was nil.